bitnami/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: elasticsearch
-version: 12.3.6
+version: 12.3.7
```

Comment:
This would be in any case a minor version change (i.e. 12.4.0)

Reply:
changed to 12.4.0